### PR TITLE
add sniffer for csv

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by iakov-aws** [comment]
Hi @tbuatois can u approve that?

File: cid/helpers/csv2view.py [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
+    """ escape for sql statement """
+    return re.sub('[^0-9a-zA-Z]+', character, text)
+
+
 def csv2view(input_file_name: str, name: str, output_file_name: str=None) -> None:
     """ Make an sql mapping from sql """
     logger.debug(f"input {input_file_name}")
+
+    sniffer = csv.Sniffer()
+    with open(input_file_name) as file_:
```
[the remark this answers is not in the document]
Comment (iakov-aws) [reply cid/helpers/csv2view.py:23]: agree

[the remark this answers is not in the document]
Comment (iakov-aws) [reply cid/helpers/csv2view.py:23]: done

**Comment by iakov-aws** [comment]
@tbuatois looks great. Can you approve this on GitHub?

**Review by tbuatois — APPROVED** [review]

**Pull request merged**